c/shared/source/uforead/uforead.c

```diff
@@ -2475,9 +2481,30 @@ static void setFlexListArrayValue(ufoCtx h) {
     freeValueArray(h);
 }
 
+static bool setType1HintDataV2(ufoCtx h, char* keyName, HintMask* curHintMask, xmlNodePtr cur) {
+    char* keyValue;
+    if (strEqual(keyName, "hintSetList")) {
+        parseHintSetListV2(h, cur);
```

Comment:
Are all of the uses of `cur` in this method NULL-safe? Seems like since you parsed a key at the level above already, it should always have a value, it would be easier to check whether `cur` is NULL above, deal with that condition there, and always be sure `cur` is not NULL here.